recipes/intake-avro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.0" %}
+{% set name = "intake-avro" %}
+{% set sha256 = "55c5945ce1b5be1bbf252c949dc5cadca596b06531bbd12a70061cb70c1e6df9" %}
+
+package:
+   name: intake-avro
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - jinja2
```

Comment:
Ack. I missed one thing. Why is jinja2 here? I poked around in the repo and I cannot figure out why/where it is needed.